cnd/src/network/start_swap.rs

```diff
@@ -0,0 +1,89 @@
+use crate::{
+    db::{ForSwap, Save, Swap},
+    network::comit::RemoteData,
+    protocol_spawner::{ProtocolSpawner, Spawn},
+    storage::{Load, Storage},
+    swap_protocols::{halight, herc20, LocalSwapId},
+};
+use ::comit::{
+    network::{WhatAliceLearnedFromBob, WhatBobLearnedFromAlice},
+    Protocol, Role, Side,
+};
+use chrono::offset::Utc;
+
+#[allow(clippy::too_many_arguments)]
+pub async fn start_swap(
```

Comment:
The missing piece!

Reply:
Yep, fixing that up is todays task when this merges.